### ISSUE TITLE
Deprecation: Don't override `this.options.babel`

### ISSUE THREAD

**Issue by jacobq** [salsify/ember-css-modules#64]
```
DEPRECATION: Ember CLI addons manage their own module transpilation during the `treeForAddon` processing. `ember-css-modules` (found at `/path/to/my/app/node_modules/ember-css-modules`) has overridden the `this.options.babel` options which conflicts with the addons ability to transpile its `addon/` files properly. Falling back to default babel configuration options.
```

**Comment by dfreeman** [comment]
Thanks for filing! Fixed in v0.6.5.

**Comment by jacobq** [comment]
Wow, that was fast; you're most welcome. :+1:
